tests/integration/test_execution.py

```diff
@@ -109,9 +111,9 @@ def test_ssh_and_scp(juju: jubilant.Juju):
     output = juju.ssh('snappass-test/0', 'ls', '/charm/containers')
     assert output.split() == ['redis', 'snappass']
     output = juju.ssh('snappass-test/0', 'ls', '/charm/container', container='snappass')
-    assert 'pebble' in output.split()
+    assert 'pebble.socket' in output.split()
     output = juju.ssh('snappass-test/0', 'ls', '/charm/container', container='redis')
-    assert 'pebble' in output.split()
+    assert 'pebble.socket' in output.split()
```

Comment:
In Juju 3.6 there is a "pebble" directory and a "pebble.socket" (socket) file. In Juju 2.9, there are two socket files "pebble.socket" and "pebble.socket.untrusted". It seems like the simplest change here is to look for the socket file rather than the directory.